Update docs for tokenFrom JWT feature (#325)
kyma-project/api-gateway: docs/istio-jwt.md

> This table lists all the possible parameters of an Istio jwt access strategy together with their descriptions:

Reviewer: ```suggestion
This table lists all the possible parameters of the Istio JWT access strategy together with their descriptions:
```

| Field                                                                     | Description                                                            |
|:--------------------------------------------------------------------------|:-----------------------------------------------------------------------|
| **spec.rules.accessStrategies.config.authentications**                    | List of authentication objects.                                        |
| **spec.rules.accessStrategies.config.authentications.issuer**             | Identifies the issuer that issued the JWT.                             |

> | **spec.rules.accessStrategies.config.authentications.jwksUri**            | URL of the provider’s public key set to validate signature of the JWT. |

Reviewer: ```suggestion
| **spec.rules.accessStrategies.config.authentications.jwksUri**            | URL of the provider’s public key set to validate the signature of the JWT. |
```

> | **spec.rules.accessStrategies.config.authentications.fromHeaders**        | List of headers from where the JWT token to be taken from.             |

Reviewer: ```suggestion
| **spec.rules.accessStrategies.config.authentications.fromHeaders**        | List of headers from which the JWT token is taken.             |
```

| **spec.rules.accessStrategies.config.authentications.fromHeaders.name**   | Name of the header.                                                    |

> | **spec.rules.accessStrategies.config.authentications.fromHeaders.prefix** | Prefix before the JWT header. Default is `Bearer `.                    |

Reviewer: ```suggestion
| **spec.rules.accessStrategies.config.authentications.fromHeaders.prefix** | Prefix used before the JWT header. The default is `Bearer`.                    |
```

> | **spec.rules.accessStrategies.config.authentications.fromParams**         | List of parameters from where the JWT token to be taken from.          |

Reviewer: ```suggestion
| **spec.rules.accessStrategies.config.authentications.fromParams**         | List of parameters from which the JWT token is taken.          |
```

| **spec.rules.accessStrategies.config.authorizations**                     | List of authorization objects.                                         |
| **spec.rules.accessStrategies.config.authorizations.requiredScopes**      | List of required scope values for the JWT.                             |
| **spec.rules.accessStrategies.config.authorizations.audiences**           | List of audiences required for the JWT.                                |

> >**NOTE:** Currently we support only a single `fromHeader` or a single `fromParameter`, mixture of both for a JWT issuer is also not supported.

Reviewer: ```suggestion
>**NOTE:** Currently, we support only a single `fromHeader` or a single `fromParameter`. Specifying both of these fields for a JWT issuer is not supported.
```

> When `istio` JWT Handler is enabled you can configure APIRule with Istio JWT like in the example below:

Reviewer: ```suggestion
When Istio JWT Handler is enabled, you can configure an APIRule with Istio JWT as in the following example:
```
